packages/storybook-a11y-test/README.md

```diff
@@ -49,6 +47,16 @@ The location of the built storybook `iframe.html` file.
 
 An array of storybook id's to skip.
 
+### testPages(options)
+
+#### iframePath `string`
+
+The location of the built storybook `iframe.html` file.
```

Comment:
How do you feel about asking people to start an http server instead, so the path to the iframe is always at localhost:6006?

Reply:
Feel free to make a seperate issue for this 👍